src/styles/components/_chart.scss

```diff
@@ -125,7 +125,7 @@
   position: absolute;
   top: 0;
   pointer-events: none;
-  z-index: 999;
+  z-index: 998;
 }
 
 .chart-internal-tooltip {
```

Comment:
by default, the tooltips are rendered in the top left corner of the screen and the z-index should be set to less than the header to avoid showing up over the header